src/Nest/Cluster/NodesInfo/NodeInfo.cs

```diff
@@ -8,35 +8,32 @@ namespace Nest
 	[DataContract]
 	public class NodeInfo
 	{
-		[DataMember(Name = "build_hash")]
-		public string BuildHash { get; internal set; }
+		[DataMember(Name = "name")]
+		public string Name { get; internal set; }
+
+		[DataMember(Name = "transport_address")]
+		public string TransportAddress { get; internal set; }
 
 		[DataMember(Name = "host")]
 		public string Host { get; internal set; }
 
-		[DataMember(Name = "http")]
-		public NodeInfoHttp Http { get; internal set; }
-
 		[DataMember(Name = "ip")]
 		public string Ip { get; internal set; }
 
-		[DataMember(Name = "jvm")]
-		public NodeJvmInfo Jvm { get; internal set; }
-
-		[DataMember(Name = "name")]
-		public string Name { get; internal set; }
+		[DataMember(Name = "version")]
+		public string Version { get; internal set; }
 
-		[DataMember(Name = "network")]
-		public NodeInfoNetwork Network { get; internal set; }
+		[DataMember(Name = "build_flavor")]
+		public string BuildFlavor { get; internal set; }
 
-		[DataMember(Name = "os")]
-		public NodeOperatingSystemInfo OperatingSystem { get; internal set; }
+		[DataMember(Name = "build_type")]
+		public string BuildType { get; internal set; }
 
-		[DataMember(Name = "plugins")]
-		public List<PluginStats> Plugins { get; internal set; }
+		[DataMember(Name = "build_hash")]
+		public string BuildHash { get; internal set; }
 
-		[DataMember(Name = "process")]
-		public NodeProcessInfo Process { get; internal set; }
+		[DataMember(Name = "total_indexing_buffer")]
+		public long TotalIndexingBuffer { get; internal set; }
```

Comment:
is this a new property in 7.7.0? I think we should add an XML comment to say which versions it's relevant to, if not relevant for all 7.x versions.

Reply:
Not sure, I will make it nullable.